README.md

```diff
@@ -2,7 +2,7 @@
 
 > :open_hands: Ever had a cool idea to an Open Source project but didn't have the time to implement yourself? Let someone else give it a try!
 
-See all ideas in the [issues list](https://github.com/mikaelbr/open-source-ideas/issues) or on the project's homepage: http://open-source-ideas.github.io/open-source-ideas/.
+See all ideas in the [issues list](https://github.com/mikaelbr/open-source-ideas/issues) or on the project's homepage: https://open-source-ideas.github.io/open-source-ideas/.
```

Comment:
Thank you. Could you also change the name `mikaelbr` -> `open-source-ideas` of the repo: as you see it was transfered to an org.